scripts/Word Simirarity Assessment/SpearmanCorrelation.py

```diff
@@ -0,0 +1,31 @@
+# Compute Spearman’s rank correlation coefficient between this score and the human judgments
+# by Aamin Abbasi
+
+import pandas as pd
+import numpy as np
+from scipy.stats import spearmanr
+
+# get dataset name
+filename = 'SemanticSimilarityDataset.csv.xlsx'
```

Comment:
You have to scan the data/similarity folder and load any CSV file there.  Don't hard code the file name 
Also for each file you have to create a same filename output CSV